ui/app/js/controllers/rule_edit_controller.js

```diff
@@ -31,6 +31,8 @@ function ($scope, $modalInstance, CSRF, Rules, Releases, rule, signoffRequiremen
   $scope.pr_ch_options = pr_ch_options;
 
   $scope.saveChanges = function () {
+   var r = confirm("You have changed the channel value. This can have unexpected impacts\n\nClick 'OK' to Confirm The Changes");
+   if (r == true) {
```

Comment:
Since the function isn't async you can just inject:
```javascript
if (!confirm("You have changed ...")) {
  // Exit early if the user denied the confirm dialog. 
  return;
}
```